c_cxx/accuracy_tool/README.md

```diff
@@ -0,0 +1,195 @@
+# ONNX Runtime accuracy testing tool
+This tool measures the accuracy of a set of models on a given execution provider. The accuracy is computed by comparing with the expected results, which are either loaded from file or attained by running the model with the CPU execution provider.
+
+## Build instructions on Windows
+Run the following commands in a terminal to configure the CMake build. Make sure to set the `ONNXRUNTIME_ROOTDIR` option to the location of your ONNX Runtime installation. You can either [download an ONNX Runtime release package](https://github.com/microsoft/onnxruntime/releases/) or you can [build ONNX Runtime from source](https://www.onnxruntime.ai/docs/build/).
+
+```shell
+$ mkdir build
+$ cd build
+$ cmake -DCMAKE_BUILD_TYPE=Release -DONNXRUNTIME_ROOTDIR='C:\onnxruntime' ..
+```
+
+Run the following command to open the solution file with Visual Studio.
+
+```shell
+$ devenv onnxruntime_accuracy_test.sln
+```
+
+Alternatively, you can build from the terminal using msbuild:
+
+```shell
+msbuild onnxruntime_accuracy_test.sln /p:Configuration=Release
+```
+
+### Building with QNN execution provider
+To test model accuracy with the QNN execution provider, call CMake with the `QNN_SDK_ROOTDIR` option set to the location of your Qualcomm AI Engine Direct SDK (QNN SDK).
+The QNN SDK can be downloaded from https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct.
+
+```shell
+$ cd build
+$ cmake -DCMAKE_BUILD_TYPE=Release -DONNXRUNTIME_ROOTDIR='C:\onnxruntime' -DQNN_SDK_ROOTDIR='C:\Qualcomm\AIStack\QNN\2.17.0.231124' ..
+```

```

Comment:
how about provide a batch script to put all these tougher?

Reply:
Good call. I added a batch script that can also take a NuGet package.